draft-gpew-priv-ppm.md

```diff
@@ -1186,18 +1188,19 @@ with payload AggregateReq.
 After verifying the authentication tag as described in
 {{aggregate-message-auth}}, the helper processes the AggregateReq as follows. It
 first scans the `AggregateReq` to check for any reports that may have been
-dropped by the leader in the previous step. Any Transition message that appears
-out of order or with has an unrecognized nonce MUST be ignored.
```

Comment:
Let's say we are running a two round VDAF. In round 1, leader sends helper an `AggregateInitReq` over five reports ordered by nonce (each composed of `(time, rand)`) thus: `(1,1), (2,2), (3,3), (4,4), (5,5)`. Upon receipt, the helper extracts its prepare message and records that it has seen those nonces. The leader can verify that the `Transitions` in the `AggregateResp` it gets appear in the same order. So far so good.

Suppose in the next round, the leader sends the next flight of prepare messages in a different order: `(5,5), (4,4), (3,3), (2,2), (1,1)`. How is helper supposed to know that this is "out of order" with respect to the previous round? Besides keeping track of whether or not it has ever seen an individual nonce, helper would have to keep track of the order in which they ever appeared in an `AggregateInitReq`. Is that reasonable? Instead of letting the leader dictate the order of reports/transitions, should we define a canonical order for reports and transitions (i.e., ascending order by nonce) so that both aggregators can statelessly enforce the desired ordering?